src/css-grids/grids.html

```diff
@@ -0,0 +1,62 @@
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>CSS Grids</title>
+  <link rel="stylesheet" href="styles.css">
+  <link href="https://fonts.googleapis.com/css?family=Hind:700&display=swap" rel="stylesheet">
+  <link href="https://fonts.googleapis.com/css?family=Work+Sans&display=swap" rel="stylesheet">
```

Comment:
move these font links above your styles.css file. you want the fonts/icons/images to load before your stylesheet can use them. so you're style sheet should be the last thing up here.

Reply:
and make it `./styles.css` rather than just `styles.css`. it's just good practice.